js/src/tooltip.js

```diff
@@ -462,7 +462,7 @@ class Tooltip {
           element.appendChild(content)
         }
       } else {
-        element.innerText = content.textContent
+        element.textContent = content.textContent
```

Comment:
This we should backport in v4 if we agree on it.

Reply:
In v4 we use `.text` of jQuery which is pretty much the same

Reply:
True that.